modules/rostests/winetests/ws2_32/sock.c

```diff
@@ -11620,9 +11631,13 @@ START_TEST( sock )
     test_GetAddrInfoW();
     test_GetAddrInfoExW();
     test_getaddrinfo();
+
+#ifndef __REACTOS__    /* ROSTESTS-385 */
+    /* UNIMPLEMENTED in ReactOS See dll/win32/msafd/misc/stubs.c */
     test_AcceptEx();
```

Comment:
We usually skip these using `if (!winetest_interactive)`, that way you also don't need to ifdef out the functions

Reply:
Changed. Thanks.